### ISSUE TITLE
[Feature Request] UX: wording when updating outdated config is confusing

### ISSUE THREAD

**Issue by sandersantema** [ampache/ampache#2913]
The following is what pops up when your config is outdated:
![image](https://user-images.githubusercontent.com/21057514/117316973-613be580-ae89-11eb-9ada-f2bf2bc10855.png)

I think the wording `Write a new config file directly to disk` is confusing, to me this suggests the config file would be written to the config folder with for instance a filename such as `ampache.cfg.php.dist` instead the old file is overwritten. Given that this is a destructive operation I think this should be more clear for instance `Overwrite old config file`. Furthermore I think it would be helpful to be able to indeed write this new config file as `ampache.cfg.php.dist` to the config folder so the user can easily for instance diff the old and new config files.

**Comment by lachlan-00** [comment]
what about a bit more explanation?
```
Your Ampache config file is out of date!
Update your current config file automatically | Download a copy of the new version
```

**Comment by sandersantema** [comment]
Wouldn't `Update` suggest the previous settings would be preserved? I.e. some sort of merge behavior, contrary to the destructive overwrite behavior which I've experienced. I don't think `Update` is clear enough in explicitly explaining a destructive operation will take place. For me at least updating implies a non destructive operation such as updating ampache, the old software is updated but user data is preserved.

What I should have mentioned is that I'm running Ampache from `https://github.com/ampache/ampache/archive/develop.tar.gz`, which I assume is the develop branch? (This isn't completely clear to me) So maybe the behavior has changed in this version.

**Comment by lachlan-00** [comment]
What part of your config is destroyed? That is a bug because it should only add the new parts

**Comment by wagnered** [comment]
The existing switches haven't been destroyed, but the settings for those switches have: Isn't the entire file overwritten by ampache.cfg.php.dist?

@sandersantema  has a point,  It's one of my pet peeves.  The request is valid: There are composer package for editing .ini files,. Laravel uses one to affect installation.  Ampache already edits the ini on setup.  It's just a matter of priorities.

**Comment by sandersantema** [comment]
Ok, this is weird. I thought I'd try and reproduce this just to be sure, but now that I did it seems to work exactly as I believe it was meant to work (I tried it thrice for good measure). What I did was to simply change the config version and try it out. 

In the case that it works the new text @lachlan-00 proposed makes a lot of sense:

> ```
> Your Ampache config file is out of date!
> Update your current config file automatically | Download a copy of the new version
> ```

So now I think it might've been a mistake on my side. It might be a good idea to create a backup of the config non the less in case it was indeed a freak accident instead of a mistake on my side. 

On another note now that I'm here, is `stream_length = "7200"` or `stream_length = 7200` correct? `"7200"` is what is written when I click the button in the interface while `7200` is what's in the `ampache.cfg.php.dist`. Is this something which is just some old leftover or should I open another issue (both values work either way)?

On even another note, I just noticed what a great config file ampache has actually got, it's very nicely documented!
